### PR TITLE
fix #188801

### LINKED ISSUE #188801: Troubleshoot issue commands casing
I think we uppercase most words in actions, except for things like "the". Here I think I would uppercase the `reproduce` in both:

image

### PR DESCRIPTION
**Pull request by sandy081**
fix #188801

### CODE REVIEW

File: src/vs/workbench/services/issue/browser/issueTroubleshoot.ts [hunk 1]
```diff
@@ -204,11 +204,11 @@ class TroubleshootIssueService extends Disposable implements ITroubleshootIssueS
 	private askToReproduceIssue(message: string): Promise<TroubleShootResult> {
 		return new Promise((c, e) => {
 			const goodPrompt: IPromptChoice = {
-				label: localize('I cannot reproduce', "I can't reproduce"),
+				label: localize('I cannot reproduce', "I can't Reproduce"),
```
Comment (alexr00) [inline src/vs/workbench/services/issue/browser/issueTroubleshoot.ts:207]: This seems weird. I think it should either be "I Can't Reproduce" or "I can't reproduce". I prefer the second. I think we only capitalize the first letter of each word if it's a command.

Comment (sandy081) [reply src/vs/workbench/services/issue/browser/issueTroubleshoot.ts:207]: @bpasero This is suggested by @bpasero. I also liked **I can't reproduce**.

Comment (bpasero) [reply src/vs/workbench/services/issue/browser/issueTroubleshoot.ts:207]: This is for buttons where we have consistent title casing:

image

Using https://titlecaseconverter.com/ shows that `can't` and `cannot` should also be uppercase 👍

Comment (sandy081) [reply src/vs/workbench/services/issue/browser/issueTroubleshoot.ts]: Wow, there is tool for this. Thanks.

Updated.

**Review by bpasero — APPROVED** [review]

**Pull request merged**